docs/kratos/guides/setting-up-password-hashing-parameters.md

```diff
@@ -18,7 +18,7 @@ security best practices, run the CLI helper included in Ory Kratos:
 kratos hashers argon2 calibrate 1s
 ```
 
```

Comment:
This file still has the `.md` extension. Please change to `.mdx`.

Reply:
@tomekpapiernik `mdx` indicates markdown + extended / react. Since there are no react components in this file, it isn't neccessary to rename it.